ReadMe.md

```diff
@@ -4,27 +4,34 @@ A Python REPL that lets you make calls to the Dropbox API v2.
 
 1. Get a Dropbox API access token.  You can use the Dropbox website to [get an access token for your own account](https://blogs.dropbox.com/developers/2014/05/generate-an-access-token-for-your-own-account/).
 
-2. Put the access token in a file called `auth.json`:
+2. Put the access token in a file called `auth.json` in the repository directory:
 
-    ```json
```

Comment:
The indentation here is intentional.  Without it, the code snippet appears to be a top-level thing, instead of being under the relevant numbered step.

Reply:
#TIL! fixed.